Harmonise README and Landing file (#2032)
alan-turing-institute/the-turing-way: README.md

<a href="https://the-turing-way.netlify.app/welcome.html"><img src="book/website/figures/logo/logo.jpg?raw=true)" width="180" align="Right" /></a>

> # The Turing Way

Reviewer: ```suggestion
# _The Turing Way_
```

> _This README.md file in also available in Dutch ([README-Dutch](README-translated/README-Dutch.md)), French ([README-French.md](README-translated/README-French.md)), German ([README-German.md](README-translated/README-German.md)), Indonesian ([README-Indonesian](README-translated/README-Indonesian.md)), Italian ([README-Italian](README-translated/README-Italian.md)), Korean ([README-Korean](README-translated/README-Korean.md)), Portuguese ([README-Portuguese](README-translated/README-Portuguese.md)), and Spanish ([README-Spanish](README-translated/README-Spanish.md)) (listed alphabetically)._

Reviewer: ```suggestion
>_This README.md file in also available in Dutch ([README-Dutch](README-translated/README-Dutch.md)), French ([README-French.md](README-translated/README-French.md)), German ([README-German.md](README-translated/README-German.md)), Indonesian ([README-Indonesian](README-translated/README-Indonesian.md)), Italian ([README-Italian](README-translated/README-Italian.md)), Korean ([README-Korean](README-translated/README-Korean.md)), Portuguese ([README-Portuguese](README-translated/README-Portuguese.md)), and Spanish ([README-Spanish](README-translated/README-Spanish.md)) (listed alphabetically)._
```

*Total Contributors:*

<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->

> [![All Contributors](https://img.shields.io/badge/all_contributors-277-orange.svg)](#contributors)

Reviewer 2: Could this be moved into the table also?

Author: I think it looks for this particular tag as written in the file currently, which is why I have not moved it to the table.

| **Build** | [![Netlify build](https://github.com/alan-turing-institute/the-turing-way/workflows/CI/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ACI+branch%3Amaster) [![Latin Phrases](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Latin%20Phrases/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Latin+Phrases%22+branch%3Amaster) [![TestFileSizes](https://github.com/alan-turing-institute/the-turing-way/workflows/TestFileSizes/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ATestFileSizes+branch%3Amaster) [![Check for Lorem Ipsums](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Lorem%20Ipsums/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Lorem+Ipsums%22+branch%3Amaster) |
| **Cite** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |

> _The Turing Way_ is an open source project that aims to make data science and research reproducible, ethical and collaborative for everyone. 

Reviewer: ```suggestion
_The Turing Way_ is an open-source project that aims to make data science and research reproducible, ethical and collaborative for everyone. 
![The Turing Way project is a book, community, an open-source project and a culture of collaboration. This is illustrated in four types of illustrations, the first one showing the Turing Way book, the second showing how the community can grow, the fourth one showing two people collaborating on a pull request, the last one is showing a balance where reproducibility is valued more than the number of paper published .](book/website/figures/README_imgs/README_turingway.png)
```

Reviewer: I think the slide which is widely used in the TW talks is very nice to add to the README. It describes the project beautifully!

Reviewer 2: I'm not sure if there is a githubbish way that I can _suggest a change_ to an already suggested change so I'll write this out:

First, love it! I think this image is a nice way to welcome people and quickly introduce them to what is happening.

> This is illustrated in four types of illustrations

is a little difficult to read. May I suggest "This is illustrated/shown/depicted/visualised in four illustrations" instead?

There are 2 typo/issues 
- I think "fourth" should be "third"
- There is a space before the full stop at the end.

Reviewer: Thank you @da5nsy, I updated it as you suggested!

Reviewer 2: Glad my suggestions were useful :)
I included "illustrated/shown/depicted/visualised" as multiple suggested options and expected you to pick one and discard the others though! 😄

Reviewer: This is hilarious 😂..

> _The Turing Way_ involves and support diverse contributors in developing lightly opinionated guides online: <https://the-turing-way.netlify.com>.

Reviewer 2: ```suggestion
_The Turing Way_ involves and supports diverse contributors in developing lightly opinionated guides online: <https://the-turing-way.netlify.com>.
```

Our moonshot goal is to make reproducibility "too easy not to do".

> ![The Turing Way project is illustrated as a road or path with shops for different data science skills. People can go in and out with their shopping cart and pick and choose what they need.](book/website/figures/welcome.jpg)

Reviewer: ```suggestion
![This is a screenshot of the online Turing Way book. It also shows one of the Turing Way illustrations at the beginning of the book. This is a road or path with shops for different data science skills. People can go in and out with their shopping cart and pick and choose what they need.](book/website/figures/README_imgs/README_book.png)
```

Reviewer: Many people might not take the effort to click on the link to the book - having a screenshot of the book in the README (I think) can attract more readers !

| **Cite** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |

_The Turing Way_ is an open source project that aims to make data science and research reproducible, ethical and collaborative for everyone. 
![The Turing Way project is a book, community, an open-source project and a culture of collaboration. This is shown in four illustrations, the first one showing the Turing Way book, the second showing how the community can grow, the third one showing two people collaborating on a pull request, the last one is showing a balance where reproducibility is valued more than the number of papers published](book/website/figures/README_imgs/README_turingway.png).

_The Turing Way_ involves and supports diverse contributors in developing lightly opinionated guides online: <https://the-turing-way.netlify.com>.
Our moonshot goal is to make reproducibility "too easy not to do".

> ![This is a screenshot of the online Turing Way book. It also shows one of the Turing Way illustrations at the beginning of the book. In this illustration, there is a road or path with shops for different data science skills. People can go in and out with their shopping cart and pick and choose what they need.](book/website/figures/README_imgs/README_book.png)

Author: ```suggestion
![This is a screenshot of the online Turing Way book. It also shows one of the Turing Way illustrations at the beginning of the book. In this illustration, there is a road or path with shops for different data science skills. People can go in and out with their shopping cart and pick and choose what they need.](book/website/figures/README_imgs/README_book.png)

*[Click use this image in a presentation](https://drive.google.com/file/d/1wJR664YECSc8b_RSHeyVjDlHs-Ls9lkc/view?usp=sharing)*
```

Author: @BatoolMM would think link to the slide be useful?

Reviewer: Incredible!

| **Build** | [![Netlify build](https://github.com/alan-turing-institute/the-turing-way/workflows/CI/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ACI+branch%3Amaster) [![Latin Phrases](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Latin%20Phrases/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Latin+Phrases%22+branch%3Amaster) [![TestFileSizes](https://github.com/alan-turing-institute/the-turing-way/workflows/TestFileSizes/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ATestFileSizes+branch%3Amaster) [![Check for Lorem Ipsums](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Lorem%20Ipsums/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Lorem+Ipsums%22+branch%3Amaster) |
| **Cite** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |

> *Welcome to The Turing Way project GitHub repository. This is where all the components of the project are developed, reviewed and maintened.*

Author: ```suggestion
*Welcome to The Turing Way project GitHub repository. This is where all the components of the project are developed, reviewed and maintained.*
```

_The Turing Way_ is a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do" even for people who have never worked in this way before.
It will include training material on version control, analysis testing, and open and transparent communication with future users, and build on Turing Institute case studies and workshops.
This project is openly developed and any and all questions, comments and recommendations are welcome at our GitHub repository: <https://github.com/alan-turing-institute/the-turing-way>.

### The Team

_The Turing Way_ a community-led project.
Everyone who contributes to this book, no matter how small or big their contributions are, is recognised in this project as a contributor and a community member. 
Long-term contributors of the project are considered part of the core contributors groups who take on various leadership roles in the project.

The project is coordinated by the lead investigators **Kirstie Whitaker** (founder) and **Malvika Sharan** (community developer) and hosted at [The Alan Turing Institute](https://www.turing.ac.uk/).

> You can read [_The Turing Way_ acknowledgement process](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) and [Record of Contributions](https://the-turing-way.netlify.app/afterword/contributors-record.html) to learn about how we acknowledge your work and how our contributors are highlighted in the project.

Reviewer 2: Is there a reason you've chosen to not use the [jupyter book cross-referencing as recommended in the style guide](https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html)?

```suggestion
You can read [_The Turing Way_ acknowledgement process](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) and {ref}`contributors-record-highlights` to learn about how we acknowledge your work and how our contributors are highlighted in the project.
```

I notice that the first link doesn't have a label, but that could (should?) be fixed.

Author: The cross referencing will not work on the README file as it is not part of the book.

Reviewer 2: Of course, silly me!

You can see the [Contributors Table](#contributors) to see the GitHub profiles of all our contributors.

### Contributing

> 

Reviewer: ![There are six illustrations to show six types of contributions that anyone can make. These are 1)developed and share, Maintain and improve, share resources, review and update, make it global through translation, share best practices](book/website/figures/README_imgs/README_contributions.png)

Reviewer: The contributing guidelines are VERY good but very long - I think having something very brief added to the README can be powerful in attracting more contributors. I added the slides about the six types of contributions you can make!  
BTW: you can always make up the same slide by combining images in Markdown but it will take me time to polish it exactly as the slide so I just exported the slides as images and added them in a new folder `README_imgs`

Author: These are amazing additions. I think we can integrate the introductory 5 minutes slides using HTML (iframe) and keep this as the first slide?

Reviewer: Thank you Malvika, do you think it's better if the slides have a dark or white background?

Author: I think white background will work well in this case.

Author: I have changed your comment as a suggestion. Please go ahead and commit this when you are ready.

Author: Unfortunately, the iframe option for sharing [slides](https://docs.google.com/presentation/d/e/2PACX-1vSK-zDTKraDEbdMBJG-XHmMLdkcfKMR9v4Hca1tXkQwdqsWidLQd8LNArUy-xjM0X8sdAeaR90Q2-zD/pub?start=false&loop=false&delayms=3000) did not work on this markdown file (works on HackMD which made me think that it will work here as well). I am ok to not have the slides. Sharing the tag in case you know any trick:
```
<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vSK-zDTKraDEbdMBJG-XHmMLdkcfKMR9v4Hca1tXkQwdqsWidLQd8LNArUy-xjM0X8sdAeaR90Q2-zD/embed?start=false&loop=false&delayms=3000" frameborder="0" width="960" height="569" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```

Reviewer: Interesting! From GitHub [specs](https://github.github.com/gfm/#example-630), it says iframe is one of the disallowed Raw HTML (extension) in GitHub flavoured Markdown (GFM). 
> the following HTML tags will be filtered when rendering HTML output:
> <title><textarea><style><xmp><iframe>

DOIs allow us to archive the repository and they are really valuable to ensure that the work is tracked in academic publications.

The citation will look something like this:

> The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. <http://doi.org/10.5281/zenodo.3233986>

You can also share the human-readable URL to a page in the book, for example, <https://the-turing-way.netlify.com/reproducibility/03/definitions.html>, but be aware that the project is under development and therefore these links may change over time.
You might want to include a web archive link such as <https://web.archive.org/web/20191030093753/https://the-turing-way.netlify.com/reproducibility/03/definitions.html> to make sure that you don't end up with broken links everywhere!

We really appreciate any references that you make to _The Turing Way_ project in your and we hope it is useful.
If you have any questions please [get in touch](#get-in-touch).

### Citing _The Turing Way_ Illustrations

> ![This is an example of one of The Turing Way illustrations. It tries to shows the evolution towards an open science era](book/website/figures/evolution-open-research.jpg)

Reviewer: ```suggestion
![This is an example of one of The Turing Way illustrations. It tries to show the evolution towards an open science era](book/website/figures/evolution-open-research.jpg)
```

> <img src="book/website/figures/evolution-open-research.jpg" alt="This is an example of one of The Turing Way illustrations. It tries to shows the evolution towards an open science era" width="600"/>

Reviewer: ```suggestion
<img src="book/website/figures/evolution-open-research.jpg" alt="This is an example of one of The Turing Way illustrations. It tries to shows the evolution towards an open science era" width="600" class="center"/>
```

DOIs allow us to archive the repository and they are really valuable to ensure that the work is tracked in academic publications.

The citation will look something like this:

> The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. <http://doi.org/10.5281/zenodo.3233986>

You can also share the human-readable URL to a page in the book, for example, <https://the-turing-way.netlify.com/reproducibility/03/definitions.html>, but be aware that the project is under development and therefore these links may change over time.
You might want to include a web archive link such as <https://web.archive.org/web/20191030093753/https://the-turing-way.netlify.com/reproducibility/03/definitions.html> to make sure that you don't end up with broken links everywhere!

We really appreciate any references that you make to _The Turing Way_ project in your and we hope it is useful.
If you have any questions please [get in touch](#get-in-touch).

### Citing _The Turing Way_ Illustrations

> 

Reviewer: ![This is an example of one of The Turing Way illustrations. It tries to shows the evolution towards an open science era](book/website/figures/evolution-open-research.jpg)

Reviewer: It might be good to add at least one example of the illustrations under this section of "Citing _The Turing Way_ illustrations".

Author: I have changed your comment to suggestion, please commit when you are ready.
